src/base/bittorrent/session.h

```diff
@@ -285,6 +285,7 @@ namespace BitTorrent
         void torrentsUpdated();
         void addTorrentFailed(const QString &error);
         void torrentAdded(BitTorrent::TorrentHandle *const torrent);
+        void torrentNew(BitTorrent::TorrentHandle *const torrent);
```

Comment:
Thanks for input but I'll keep it as is.
`torrentNew` to follow the naming scheme of the other signals. Not renaming the old signal, in order to maintain the same meaning as libtorrent.